lib/service/connect.go

```diff
@@ -590,17 +590,23 @@ func (process *TeleportProcess) firstTimeConnect(role types.SystemRole) (*Connec
 			return nil, trace.Wrap(err)
 		}
 
+		dataDir := defaults.DataDir
```

Comment:
Was this an unrelated bug?

Reply:
I believe so. I've only encountered this when running integration tests - my `DataDir` was a temp directory, and it was failing to read certificates from `/var/lib/teleport`. Changing this fixed it, and I can't imagine a scenario where you'd ever want your CAs to live in `/var/lib/teleport` and `DataDir` be set to something else, and have no configuration over the matter.

Reply:
This might be something worth fixing in v10 and v9. Let's not lose track of it since this PR won't be backported.